File: BackEnd/src/main/java/com/example/backend/services/LessonsService.java
package com.example.backend.services;

import com.example.backend.baseClasses.BaseCRUDService;
import com.example.backend.data.models.Lesson;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Service;

@Service
public class LessonsService extends BaseCRUDService<Lesson, Integer> {

    public LessonsService(CrudRepository<Lesson, Integer> repository) {
        super(repository);
    }
}

File: TelegramBot/TelegramBot/Services/ApiServices/LessonsService.cs
﻿using TelegramBot.Data.Entities;
using TgBotLib.Services;

namespace TelegramBot.Services.ApiServices
{
    public class LessonsService : BaseCRUDService<Lesson, int>
    {
        public LessonsService() : base(AppSettings.LessonsRoot)
        {
        }
    }
}